tests/integration/connections/debug_envgetter.py

```diff
@@ -0,0 +1,24 @@
+import os
+import json
+import logging
+
+
+class EnvGetter:
```

Comment:
Maybe rename to `TestEnvGetter` and inherit `EnvGetter` ? (it's very confusing to have 2 classes with the exact same name)

Reply:
sure will do

Reply:
Done